Mathlib/Algebra/Module/Submodule/Pointwise.lean

```diff
@@ -281,4 +284,220 @@ scoped[Pointwise] attribute [instance] Submodule.pointwiseMulActionWithZero
 
 end
 
+section
```

Comment:
Can you name this section, and update the "#implementation notes" in the module docstring to make it clear that the lemmas from this point on are not "direct copies of lemmas from
`GroupTheory/Submonoid/Pointwise.lean`."?

Reply:
done